components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/RDBMSSessionDataStore.java

```diff
@@ -46,56 +47,35 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.LinkedBlockingDeque;
 
-/**
- * Data will be persisted or stored date will be removed from the store. These two events are considered as STORE operation
- * and DELETE operations.
- * And these events are stored with unique sessionId, operation type and operation initiated timestamp.
- * Expired DELETE operations and related STORE operations will be deleted by a OperationCleanUpService task.
- * All expired operations will be deleted by SessionCleanUpService task.
- */
 public class RDBMSSessionDataStore extends SessionDataStore {
 
     public static final String DEFAULT_SESSION_STORE_TABLE_NAME = "IDN_AUTH_SESSION_STORE";
     public static final String DEFAULT_TEMP_SESSION_STORE_TABLE_NAME = "IDN_AUTH_TEMP_SESSION_STORE";
-    public static final BlockingDeque<SessionContextDO> sessionContextQueue = new LinkedBlockingDeque();
     private static final Log log = LogFactory.getLog(SessionDataStore.class);
-    private static final String OPERATION_DELETE = "DELETE";
-    private static final String OPERATION_STORE = "STORE";
-    private static final String SQL_INSERT_STORE_OPERATION =
-            "INSERT INTO IDN_AUTH_SESSION_STORE(SESSION_ID, SESSION_TYPE, OPERATION, SESSION_OBJECT, TIME_CREATED, EXPIRY_TIME, TENANT_ID) VALUES (?,?,?,?,?,?,?)";
-    private static final String SQL_INSERT_DELETE_OPERATION =
-            "INSERT INTO IDN_AUTH_SESSION_STORE(SESSION_ID, SESSION_TYPE,OPERATION, TIME_CREATED, EXPIRY_TIME) VALUES (?,?,?,?,?)";
-    private static final String SQL_DELETE_STORE_OPERATIONS_TASK =
-            "DELETE FROM IDN_AUTH_SESSION_STORE WHERE OPERATION = '" + OPERATION_STORE + "' AND SESSION_ID in (" +
-                    "SELECT SESSION_ID  FROM IDN_AUTH_SESSION_STORE WHERE OPERATION = '" + OPERATION_DELETE + "')";
-    private static final String SQL_DELETE_TEMP_STORE_OPERATIONS_TASK =
-            "DELETE FROM IDN_AUTH_TEMP_SESSION_STORE WHERE EXPIRY_TIME < ?";
-    private static final String SQL_DELETE_STORE_OPERATIONS_TASK_MYSQL =
-            "DELETE IDN_AUTH_SESSION_STORE_DELETE FROM IDN_AUTH_SESSION_STORE IDN_AUTH_SESSION_STORE_DELETE WHERE " +
-                    "OPERATION = '" + OPERATION_STORE + "' AND SESSION_ID IN (SELECT SESSION_ID FROM (SELECT SESSION_ID " +
-                    "FROM IDN_AUTH_SESSION_STORE WHERE OPERATION = '" + OPERATION_DELETE + "') " +
-                    "IDN_AUTH_SESSION_STORE_SELECT)";
-    private static final String SQL_DELETE_DELETE_OPERATIONS_TASK =
-            "DELETE FROM IDN_AUTH_SESSION_STORE WHERE OPERATION = '" + OPERATION_DELETE + "' AND  EXPIRY_TIME < ?";
+    private static final String SQL_INSERT =
+            "INSERT INTO IDN_AUTH_SESSION_STORE(SESSION_ID, SESSION_TYPE, SESSION_OBJECT,TIME_CREATED, EXPIRY_TIME, TENANT_ID) " +
+                    "VALUES " + "(?,?,?,?,?,?)";
+    private static final String SQL_DELETE =
```

Comment:
I feel like having the SQL query constants here makes the class much longer in terms of the line count. Shall we move all the constants to a static class in another file?

Reply:
To the framework constant class?